tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -14,7 +14,6 @@
 class Homescreen(Base):
 
     name = 'Default Home Screen'
-    manifest_url = '{}verticalhome{}/manifest.webapp'.format(Base.DEFAULT_PROTOCOL,Base.DEFAULT_APP_HOSTNAME)
 
```

Comment:
Just to confirm, even with the manifest_url gone, it'll still work with non-English locale right?